src/components/common/needsShutdown.tsx

```diff
@@ -152,6 +197,14 @@ export function getDevicesRequiringShutdown(vm: VM) {
     if (needsShutdownSpice(vm))
         devices.push(_("SPICE"));
 
+    // VNC
+    if (needsShutdownVnc(vm))
+        devices.push(_("VNC"));
+
+    // Serial console
+    if (needsShutdownSerialConsole(vm))
+        devices.push(_("Text console"));
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2008-57844738-20250716-080045-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)